dvc/exceptions.py

```diff
@@ -309,3 +309,10 @@ def __init__(self, path, repo):
             " neighther as an output nor a git-handled file."
         )
         super().__init__(msg.format(path, repo))
+
+
+class UpdateWithRevNotPossibleError(DvcException):
+    def __init__(self):
+        super(UpdateWithRevNotPossibleError, self).__init__(
+            "Revision option (`--rev`) is not a feature of non-Git sources."
```

Comment:
Usually it is easier to read without _double negations_:
`not a feature of non-Git sources` -> `only a feature of Git sources`

```suggestion
            "Revision option (`--rev`) is supported only for Git sources."
```